src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int getRandomNumber() {
+        return new Random().nextInt(100);
+    }
+
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        Ball ball = new Ball();
+        Colors randomColor = color.getRandomColor();
+        ball.setColor(randomColor);
```

Comment:
```suggestion
public class Lottery {
    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random random = new Random();
    
    private int getRandomNumber() {
        return random.nextInt(100);
    }

    public Ball getRandomBall() {
        Ball ball = new Ball();
        Colors randomColor = colorSupplier.getRandomColor();
        ball.setColor(randomColor);
```
<img width="1115" alt="image" src="https://github.com/mate-academy/jv-lottery/assets/81781479/b6a66244-ca8f-46ee-ad00-78f1470e8d14">